pkg/kubelet/rkt/rkt.go

```diff
@@ -246,6 +249,22 @@ func New(
 		requestTimeout = defaultRequestTimeout
 	}
 
+	imgSvc, err := rktshim.NewImageStore(rktshim.ImageStoreConfig{
+		RequestTimeout: requestTimeout,
+		// TODO(tmrts): use the new CLI api throught the rkt pkg
+		CLI: rktshim.NewRktCLI(config.Path, execer, rktshim.CLIConfig{
+			Debug:           config.Debug,
```

Comment:
Hmm, this only takes the config passed by kubelet, however the semantics is if `rkt api service` specifies the config, it will override the config passed by kubelet.
See below `rkt.config, err = rkt.getConfig(rkt.config)`